scico/admm.py

```diff
@@ -170,11 +170,14 @@ def internal_init(self, admm):
         if admm.f is not None:
             if not isinstance(admm.f.A, LinearOperator):
                 raise ValueError(
-                    "LinearSubproblemSolver requires f.A to be a scico.linop.LinearOperator; "
+                    f"LinearSubproblemSolver requires f.A to be a scico.linop.LinearOperator; "
                     f"got {type(admm.f.A)}"
```

Comment:
To nitpick, I don't think the `f` is necessary here.

Reply:
You are right. I added it though to avoid mixing stings and f-strings. (Similar to preferring `2.0 * np.pi` to `2 * np.pi` where the `.0` is not necessary.)

Feeling strongly about removing this?

Reply:
No strong feelings. I primarily mentioned it because I seemed to recall that `flake8` or `pylint` complained about it, but on checking that doesn't seem to be the case, so consider the comment withdrawn.